### PR TITLE
Add ci-kubernetes-cosbeta-*-soak test jobs

### PR DESCRIPTION
**Pull request by yguo0905**
I don't expect those settings are correct in one shot - might need to adjust later. But it would be great if you can double check them for me.

/assign @krzyzacy 
/cc @abgworrall, @dchen1107, @yujuhong

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -1414,6 +1414,36 @@ test_groups:
   - configuration_value: master_os_image
   - configuration_value: Commit
   - configuration_value: infra-commit
+- name: ci-kubernetes-e2e-gke-cosbeta-k8sdev-soak
+  gcs_prefix: kubernetes-jenkins/soak/ci-kubernetes-e2e-gke-cosbeta-k8sdev-soak
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: the log path is still `k-j/logs/job-name`, the soak bucket is for cluster credentials

File: testgrid/jenkins_verify/jenkins_validate.go [hunk 2]
```diff
@@ -102,7 +102,8 @@ func main() {
 	// For now anything outsite k8s-jenkins/(pr-)logs are considered to be fine
 	testgroups := make(map[string]bool)
 	for _, testgroup := range config.TestGroups {
-		if strings.Contains(testgroup.GcsPrefix, "kubernetes-jenkins/logs/") {
+		if strings.Contains(testgroup.GcsPrefix, "kubernetes-jenkins/logs/") ||
+			strings.Contains(testgroup.GcsPrefix, "kubernetes-jenkins/soak/") {
```
Comment (krzyzacy) [inline testgrid/jenkins_verify/jenkins_validate.go]: and you won't need this change :-)

File: testgrid/config/config.yaml (hunk 1 above)
Comment (yguo0905) [reply testgrid/config/config.yaml]: Oh, the `--save` in the job definition is for creds.

**Comment by yguo0905** [comment]
PTAL!

**Pull request merged**